ros/angel_system_nodes/setup.py

```diff
@@ -28,7 +28,8 @@
                 'spatial_mapper = angel_system_nodes.spatial_mapper:main',
                 'activity_detector = angel_system_nodes.activity_detector:main',
                 'task_monitor = angel_system_nodes.task_monitor:main',
-                'feedback_generator = angel_system_nodes.feedback_generator:main'
+                'feedback_generator = angel_system_nodes.feedback_generator:main',
+                'descriptor_generator = angel_system_nodes.object_descriptor_generator:main'
```

Comment:
Might as well end the line with a comma so that we don't have to remember to modify the line when we add something new in the future.